libs/database/src/lib/repositories/index.ts

```diff
@@ -627,7 +634,6 @@ export class Repository {
 
     return new this.userModel(user).save();
   }
-
   private getUserQuery(filter: UserQuery) {
```

Comment:
```suggestion

  private getUserQuery(filter: UserQuery) {
```
อยากให้มี new line ระหว่าง function+function, class+class, import+class + ก่อน import พอมันติดกันหมด มันอ่านยากครับ.